mapbox_gl_web/lib/mapbox_gl_web.dart

```diff
@@ -0,0 +1,26 @@
+library mapbox_gl_web;
+
+import 'dart:async';
+// FIXED HERE: https://github.com/dart-lang/linter/pull/1985
+// ignore_for_file: avoid_web_libraries_in_flutter
+import 'dart:html';
+import 'dart:js';
+import 'dart:ui' as ui;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+import 'package:mapbox_gl_platform_interface/mapbox_gl_platform_interface.dart';
+import 'package:mapbox_gl_dart/mapbox_gl_dart.dart' hide Point;
+import 'package:mapbox_gl_dart/mapbox_gl_dart.dart' as mapbox show Point;
```

Comment:
What is the reasoning behind this construction?

Reply:
I think it's because there are Point classes both in the mapbox_gl_dart package from pub and builtin to flutter. These import statements require a 'mapbox.'-prefix only when using the Point class fom mapbox_gl_dart, but no prefix when using any other classes from that package or the builtin Point class.

Reply:
Got it. I would have hidden the Point from the builtin Flutter package but this also works.

Reply:
I don't hide Flutter version because we use it! see `queryRenderedFeatures`